### DISCUSSION TITLE
v25.0.0-beta.1

### DISCUSSION THREAD

**Discussion by neersighted** [moby/moby#46816] [Announcements]
## 25.0.0-beta.1

This is a pre-release of the upcoming 25.0.0 release.

Pre-releases are intended for testing new releases: **only install in a test environment!**

```bash
curl -fsSL https://get.docker.com -o get-docker.sh
sudo CHANNEL=test sh get-docker.sh
```

### Known issues:

- There is no changelog yet; an overview of pull requests included in this release can be found on GitHub:
    - docker cli: [all pull requests for 25.0.0](https://github.com/docker/cli/pulls?q=is%3Apr+milestone%3A25.0.0+is%3Amerged) / [all "changelog" pull requests for 25.0.0](https://github.com/docker/cli/pulls?q=is%3Apr+milestone%3A25.0.0+label%3Aimpact%2Fchangelog+is%3Amerged)
    - docker engine: [all pull requests for 25.0.0](https://github.com/moby/moby/pulls?q=is%3Apr+milestone%3A25.0.0+is%3Amerged) / [all "changelog" pull requests for 25.0.0](https://github.com/moby/moby/pulls?q=is%3Apr+milestone%3A25.0.0+label%3Aimpact%2Fchangelog+is%3Amerged)
- dockerd now uses systemd's default `LimitNOFILE` which on older versions of systemd, such as used by CentOS 7 is very low and may limit the number of containers that can be run. Set `LimitNOFILE=1048576` to get the previous behavior.
- There are no packages available yet for the s390x and ppc64le architectures
- There may be bugs! :warning:

Bugs and regressions can be reported in these issue trackers:

- Related to the CLI: https://github.com/docker/cli/issues
- Related to the Docker Engine https://github.com/moby/moby/issues

When reporting issues, include `[25.0.0-beta]` in the issue title

<hr /><em>This discussion was created from the release <a href='https://github.com/moby/moby/releases/tag/v25.0.0-beta.1'>v25.0.0-beta.1</a>.</em>

**Comment by rst0git** [comment]
After upgrading to version 25.0.0-beta.1 checkpoint/restore fails with the following error:

```
$ docker start --checkpoint=c1 cr
Error response from daemon: failed to create task for container: content digest fdb1054b00a8c07f08574ce52198c5501d1f552b6a5fb46105c688c70a9acb45: not found: unknown
```
This functionality works with version 24.0.7
